### ISSUE TITLE
The compression package of the pre-trained model is broken

### ISSUE THREAD

**Issue by tuyaxin** [NVIDIA/flownet2-pytorch#187]
I got  fault when decompressing the pre-trainied model. Please help !!!
Anyone had this resolved, please help! Thanks a lot.

**Comment by trungmanhhuynh** [comment]
I don't think we have to decompress it. Should it be checkpoints/*.pth.tar ? The code is reading it fine.
